### ISSUE TITLE
Atomic Lamp can't be activated

### ISSUE THREAD

**Issue by FredZeppelin** [CleverRaven/Cataclysm-DDA#11830]
Should the atomic lamp be something you can turn on and off? It doesn't seem to be able to be activated...

**Comment by FireCrow1973** [comment]
(when i last got one) it gives a light light to the adjacent squares.
The atomic core glows all the time, just put it on the ground and get cosy!

**Comment by FredZeppelin** [comment]
Ok gotcha. I guess I expected it to function like a flashlight, or how a lamp in real life would - with an on/off switch.

**Comment by Shoes01** [comment]
It's the "atomic trade off", lasts forever, can't be turned off.

**Comment by Headjack** [comment]
I really wish there was a shade or something you could just use to block the light. Don't wanna glow all the time for Zed.

**Comment by sparr** [comment]
@Headjack if it had a shade, it would be strictly superior to the flashlight and other activateable lights. As it is now, they both have pros/cons.

**Comment by Rivet-the-Zombie** [comment]
They're super heavy for their size, and you can't turn them off, but they last forever and a day. If we added the ability to turn them off, they would be far too good, and superior to all other forms of portable lighting. So that's not happening.

*tl;dr* No off switch, no shade.
